### DISCUSSION TITLE
Ideapad Flex 5-14IIL05 - 0.8.4 Keyboard not working after reboot fixed

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore#11] [Announcements]
### News / Fixes:
- Updated VoodooPS2Controller that adds fix for keyboard after reboot I remember that being an issue so I want to help you with this.
- Now Keyboard works always. Not tested, but the kext is the only thing updated.

### Not Working:
- Sleep is currently not working and it will reboot if you press the button from Apple Logo.
- Trackpad
- Internal Microphone (needs a different layout for AppleALC)
- Touch Display
- Finger Unlock (Never would)

### Other Issues:
- Ive noticed some TSC issues on latest boot stage might need a fix in the future

With this EFI you should be able to boot into macOS Big Sur UP to latest macOS Ventura and install it with no problem.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore/releases/tag/0.8.4v3'>Ideapad Flex 5-14IIL05 - 0.8.4 Keyboard not working after reboot fixed</a>.</em>

**Comment by rhysbw** [comment]
I might be completely missing the point. I have the exact same model as you. I have downloaded the release .zip file, I then became a bit stuck on how to make this bootable. I did some digging and thought i needed to just put the unzipped version on a USB and then boot to it with Fn+F12. But when i do this nothing seems to happen and it then just boots back into windows. Any help?

**Reply by rhysbw** [reply]
On this, i managed to make the usb, when using Monterey the install would make me connect to internet, when i would do this the system would crash (BUT the wifi was detected and could see networks). The tried with Big Sur, this progressed further, but now i do not have wifi and no wifi card is detected, do u know why this might be?

**Reply by RobyRew** [reply]
you need to change the AirPortitwl kext for the Monterrey one, also, another kext combination for bluetooth is required, its very different from what works right now on Ventura
